### DISCUSSION TITLE
How to apply for Platform Moderation/Moderator?

### DISCUSSION THREAD

**Discussion by Test1calCutter** [stoatchat/discussions#522] [Q&A] [answered]
**Hello there! 👋**
I was wondering how someone can apply for Moderator or something other. And what the requirements are if it's even possible to do so.

Thanks :D

**Comment by kentledge** [comment, marked as answer]
Hello, I had emailed them about the same thing and within a few days, I got a response saying that they arent looking for any as of right now.
